### PR TITLE
0.3.5 release notes

### PR DESCRIPTION
**Pull request by sofroniewn**
# Description
0.3.5 release notes

### CODE REVIEW

File: docs/release/release_0_3_5.md
Comment (jni) [inline docs/release/release_0_3_5.md]:
```suggestion
- Allow using of custom color dictionary in labels layer (#1339 and #1362)
```

File: docs/release/release_0_3_5.md
Comment (jni) [inline docs/release/release_0_3_5.md]: This needs a better title... 
```suggestion
- Fix Linux and Windows key hold detection (#1350)
```
?

File: docs/release/release_0_3_5.md
Comment (jni) [inline docs/release/release_0_3_5.md]: So does this... Maybe we need to do a better job of monitoring the names of PRs...

```suggestion
- Fix crash when selecting all points (#1358)
```

?

File: docs/release/release_0_3_5.md
Comment (jni) [inline docs/release/release_0_3_5.md:12]: I would add a highlights section here, something like:

```
## Highlights

This release contains a number of bug fixes on various platforms. For those interested
in napari performance, we have added a new performance monitoring mode, that can
be activated by <method + link to relevant docs>. We have also added a page on
napari's rendering pipeline [here](<link to new rendering page>)
```

Currently I can't find either of those doc links, though. I'll raise an issue.

Comment (jni) [reply docs/release/release_0_3_5.md:12]: See #1365

File: docs/release/release_0_3_5.md
Comment (VolkerH) [inline docs/release/release_0_3_5.md]: Just spotted this typo ... not that it matters much ... "ranger slide" should probably be "range slider"

Comment (sofroniewn) [reply docs/release/release_0_3_5.md]: Thanks! Fixed

File: docs/release/release_0_3_5.md
Comment (sofroniewn) [reply docs/release/release_0_3_5.md:12]: I have added this but only referenced the PRs until #1365 is fixed

**Comment by sofroniewn** [comment]
@jni this is ready for re-review

**Review by jni — APPROVED** [review]
@sofroniewn LGTM but to be honest I would prefer waiting until the docs are fixed to release...!

File: docs/release/release_0_3_5.md
Comment (kevinyamauchi) [inline docs/release/release_0_3_5.md:25]: Is #1342 being included in this release? If so, should we add it here?

```suggestion
- Add performance monitoring widget (#1262)
- Allow Shapes face and edge colors to be mapped to properties (#1342)
```

Comment (sofroniewn) [reply docs/release/release_0_3_5.md:25]: yes, not sure why that wasn't there, i guess i made release notes before merge or something. thanks for catching!!

**Comment by tlambert03** [comment]
> @sofroniewn LGTM but to be honest I would prefer waiting until the docs are fixed to release...!

@jni did you have any thoughts on https://github.com/napari/napari/issues/1365#issuecomment-644982643?

**Comment by sofroniewn** [comment]
I have now added a link to the rendering doc. I get the link for the perf mon stuff to be the PR as that is where the best explanations are. I will merge this once some tests pass

**Pull request merged**
